### ISSUE TITLE
Easy log handler not logging

### ISSUE THREAD

**Issue by TheBabaYaga** [EasyCorp/easy-log-handler#2]
I've installed the bundle and replaced the logging config in my config_dev, but now it isn't logging at all anymore. Is there a way to debug it to see what's going on?

I'm using following Symfony version:

`"name": "symfony/symfony",
            "version": "v2.8.9",
            "source": {
                "type": "git",
                "url": "https://github.com/symfony/symfony.git",
                "reference": "df02dd5d3f7decb3a05c6d0f31054b4263625dcb"
            }`

**Comment by javiereguiluz** [comment]
Let's try to debug the problem. I assume that logging is working when using the common config:

``` yaml
# app/config/config_dev.yml
monolog:
    handlers:
        main:
            type: stream
            path: "%kernel.logs_dir%/%kernel.environment%.log"
            level: debug
            channels: [!event]
        console:
            type:   console
            channels: [!event, !doctrine]
```

Now, can you replace that config by the following?

``` yaml
# app/config/config_dev.yml
monolog:
    handlers:
        buffered:
            type:     buffer
            handler:  easylog
            channels: ["!event"]
            level:    debug
        easylog:
            type: service
            id:   easycorp.easylog.handler
        main:
            type: stream
            path: "%kernel.logs_dir%/%kernel.environment%.log"
            level: debug
            channels: [!event]
        console:
            type:   console
            channels: [!event, !doctrine]
```

The `app/logs/dev.log` should now contain both the old Symfony logs and the new EasyLog logs. If not: Do you see any error in the app? Do you see any log in `dev.log`?

**Comment by TheBabaYaga** [comment]
With the second config I just see my regular default log data, but not the EasyLog logs.
![screen shot 2016-08-17 at 10 21 17](https://cloud.githubusercontent.com/assets/3961971/17729414/6a4b00a8-6464-11e6-91ed-0e58eab4ff30.png)

**Comment by TheBabaYaga** [comment]
@javiereguiluz version 1.0.1 fixed this issue and #3 as well! Thanks!
